tests/sqllogictests/suites/mode/cluster/merge_into_non_equal_distributed.test

```diff
@@ -64,29 +64,28 @@ target_table: default.default.t1
 ├── matched update: [condition: None,update set a = a (#0)]
 └── unmatched insert: [condition: None,insert into (a) values(CAST(a (#0) AS Int32 NULL))]
 HashJoin
```

Comment:
It would be better to add a test for `MergeSource::Select`.

Reply:
get